### ISSUE TITLE
train all the parameters together？

### ISSUE THREAD

**Issue by xianshunw** [twhui/LiteFlowNet#53]
In your paper, you mentioned that the network is trained stage-wised. Have you ever train all parts of your network together and how it performs? Since your work is a very important baseline for us and your training process is completed for us, could you provide us more training details? For example, how fine-tune on FlyingThings(iteration times, learning rate and loss)? 

Thanks a lot for your contributions.

**Comment by twhui** [comment]
Different networks use different hyper-training parameters. If you want to use LiteFlowNet as your basline, you can simply get the trained models here (/LiteFlowNet/models/trained/).

Fine-tuning on Things3D is similar to FlowNet2, i.e. 500k iterations, L2 loss, ...etc

**Comment by xianshunw** [comment]
thanks. closed.
